client_int_test.go

```diff
@@ -153,6 +153,8 @@ func TestRemove(t *testing.T) {
 // newClient creates an instance of the func client whose concrete impls
 // match those created by the kn func plugin CLI.
 func newClient(verbose bool) *boson.Client {
+	// TODO: Forcing verbose to false in order to pass integration tests
+	verbose = false
```

Comment:
This seems like an odd thing to have cause a failure.  Please let's chat after we're over the hump with this release; I'd like to fix the root cause

Reply:
It's indeed strange. I was able to run the test successfully on my machine with with and without verbose output.

Reply:
I was able to run it successfully only if running the test in isolation.  When in tandem with others: failure.

It looks like it has to do with the pack logger being flaky when passed a stdout-backed logger (such as in verbose mode) @matejvasek

Reply:
Furthermore, when working around that issu, it also appears to be having trouble when using the builtin logger as well.  it errors failing to connect to stderr.  @matejvasek

Reply:
I am not sure if this is a pack logger issue or what (thought it seems so), but it appears we can get around the issue by always using a bytes.Buffer-backed logger, and only printing when in verbose mode (or embedding it in any errors).  Refactoring a bit to clear things out, and submitting an update. @matejvasek

Reply:
>I was able to run it successfully only if running the test in isolation. When in tandem with others: failure.

I run the failing test in isolation.